### PR TITLE
[stable/traefik] Support acme.onDemand (#1626)

### PR DESCRIPTION
**Pull request by grugnog**
See #1626

### CODE REVIEW

**Comment by unguiculus** [comment]
cc @krancour

File: stable/traefik/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.15.2
+version: 1.15.3
```
Comment (krancour) [inline stable/traefik/Chart.yaml:3]: This is a new feature, so this should increment the minor version.

**Comment by krancour** [comment]
This PR seems totally reasonable, but Traefik has actually deprecated this feature. I'm not sure that we should be adding support in the chart for something Traefik itself is phasing out.

**Comment by unguiculus** [comment]
Should we close it then?

**Comment by grugnog** [comment]
Didn't realize this was being depreciated (the docs with the depreciation only got released a week ago I think). Just going to close this out for now.

Of note, LE wildcard certs are going live February 27 which should cover many of the use cases this would be needed for.

**Pull request closed without merging**